types/react-native-autocomplete-input/index.d.ts

```diff
@@ -78,21 +66,16 @@ export interface AutocompleteProps<T> extends TextInputProperties {
 
     /**
      * function
-     * renderItem will be called to render the data objects which will be displayed in the result view below the text input.
+     * render custom TextInput. All props passed to this function.
      */
-    renderItem(itemWithIndex: { item: T; index: number }): ReactNode;
+    renderTextInput?(props: TextInputProps): ReactNode;
 
     /**
-     * function
-     * renderSeparator will be called to render the list separators which will be displayed between the list elements in the result view below the text input.
+     * string | bool
+     * Set `keyboardShouldPersistTaps` to true if RN version is <= 0.39.
      */
-    renderSeparator?: ListViewProperties['renderSeparator'];
 
-    /**
-     * function
-     * render custom TextInput. All props passed to this function.
-     */
-    renderTextInput?(props: TextInputProperties): ReactNode;
+    keyboardShouldPersistTaps?: 'always' | 'handeld' | 'never' | boolean;
 }
 
 export default class Autocomplete<T> extends Component<AutocompleteProps<T>> {}
```

Comment:
this is now wrong IMO, if using 5.0, can you double check?:
https://github.com/mrlaessig/react-native-autocomplete-input/blob/master/index.js#L5

Reply:
What's exactly wrong?
Taking a look at: https://github.com/mrlaessig/react-native-autocomplete-input/blob/master/index.js#L79 shows  the values of `keyboardShouldPersistTaps`

Reply:
sorry to be succint on the words, the export is not a class anymore, functioinal component IMO
I didn't mean props, sry!

Reply:
Ah ok, you were talking about line 81. I got it and will fix it :)